DEVDOCS-6277 - update content for new dev portal (#915)
bigcommerce/docs: docs/integrations/apps/guide/publishing.mdx

* App summary
* App category

## Provide technical information

In the **App information** tab, provide the relevant technical information required for the app’s functionality. Specify callback URLs and multiple user support. This tab also provides the app’s API credentials.

![App information](https://storage.googleapis.com/bigcommerce-production-dev-center/Big%20Design%20Docs/next-devtools-technical-info.png)

| Field | Description |
| --- | --- |
| Auth | Requested when store owner or authorized user clicks `install` |
| Load | Requested when user launches app |
| Uninstall | Requested when store owner or authorized user clicks `uninstall` |
| Enable multiple users | Optionally allow your app to be accessible to store users other than the owner |

> | Remove user callback URL | Requested when a store owner opts to revoke a user’s authorization to use the app |

Reviewer: It would be good to either (1) add more context here or (2) link to this other doc that explains a bit more about supporting multiple users https://developer.bigcommerce.com/docs/integrations/apps/guide/users

Author: ```suggestion
| Remove user callback URL | Requested when a store owner opts to revoke a user’s authorization to use the app. For more information, see [Supporting Multiple Users](/docs/integrations/apps/guide/users) |
```

## Add OAuth Scopes

In the **Scopes** tab, add any scopes the app requires. For a list, see [OAuth scopes](/docs/start/authentication/api-accounts#oauth-scopes).

![Scopes](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/next-devtools-scopes-tight.png "App scopes")

| Field | Description |
| --- | --- |
| Set manually | Allows explicit selection of the necessary scopes |

> | Modify all | Sets all scopes to Modify or the most permissive selection |

Author: ```suggestion
| Modify all | Sets all scopes to Modify or the most permissive selection |

<Callout type="info>
"Modify All" is for testing only. Apps configured with all scopes will not be approved for public listing in the Apps Marketplace.
</Callout>
```

|Scope permission | Description |
| --- | --- |
| None | Removes app access to the scope entirely |
| Read-Only | Only grants the app read access for the scope |
| Modify | Grants the app full access to read, write, create, or delete data |
| Create | Grants the app access to create data |
| Get | Grants the app access to read specific data |
| Token Login | Allows management of login processes including authentication and session handling |

> | Generate Tokens | Allows generation of access tokens used to access storefront or customer account information |

Author: Added callout between the first table and the second. Thanks for the note.

| App screenshots | Screenshots of the app as end-users experience it. Displayed on the app’s detail page in the apps marketplace. |

### Videos, features, and case studies

Relevant videos of app functionality, features that users will find appealing, and case studies that demonstrate the app’s usefulness.

<Callout type="info">
  Videos and Case studies are optional fields.
</Callout>

![Videos, Features, and Case studies](https://storage.googleapis.com/bigcommerce-production-dev-center/Big%20Design%20Docs/next-devtools-media-assets.png  "Videos, Features, and Case studies")

| Field | Description |
| --- | --- |
| Videos | Videos highlighting purpose and value of your app |
| Case Studies | Case studies demonstrating how merchants have benefited from using your app. 4 max |
| Features | App's major features; include title and description for each; title is search indexed; rich text accepted; 5 max |

### Resources and International optimization

Useful resources and locale information for the app.

<Callout type="info">

>   Despite the indication within the editor that Company privacy policy and Company terms of service are optional, they are required for legal reasons and should be included.

Reviewer: "Despite the indication within the editor that Company privacy policy and Company terms of service are optional," we should either change the UI or make this less passive. Something like "Company privacy policy and Company terms of service are required for legal reasons and should be included. Apps without these items will not be listed in the Marketplace until this information is provided."

Author: ```suggestion
  Company privacy policy and Company terms of service are required for legal reasons and should be included. Apps without these items will not be listed in the Marketplace until this information is provided.
```

![App information](https://storage.googleapis.com/bigcommerce-production-dev-center/Big%20Design%20Docs/next-devtools-technical-info.png)

| Field | Description |
| --- | --- |
| Auth | Requested when store owner or authorized user clicks `install` |
| Load | Requested when user launches app |
| Uninstall | Requested when store owner or authorized user clicks `uninstall` |
| Enable multiple users | Optionally allow your app to be accessible to store users other than the owner |
| Remove user callback URL | Requested when a store owner opts to revoke a user’s authorization to use the app. For more information, see [Supporting Multiple Users](/docs/integrations/apps/guide/users) |

## Add OAuth Scopes

In the **Scopes** tab, add any scopes the app requires. For a list, see [OAuth scopes](/docs/start/authentication/api-accounts#oauth-scopes).

![Scopes](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/next-devtools-scopes-tight.png "App scopes")

| Field | Description |
| --- | --- |
| Set manually | Allows explicit selection of the necessary scopes |
| Modify all | Sets all scopes to Modify or the most permissive selection |

> <Callout type="info>

Author: ```suggestion
<Callout type="info">
```
